domains/measurements/Measurements.API/API/Properties/launchSettings.json

```diff
@@ -1,31 +1,31 @@
-{
```

Comment:
Why changes to this file?

Reply:
dotnet format did it. I had a --swagger to test it worked. But somehow me touching it made it change, when I changed it back.